retrofit/pom.xml

```diff
@@ -32,6 +32,11 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
```

Comment:
Oh okay. Should we have our own `MultiMap`? Or should we support `Map<String, Collection<?>>` instead?